Gifski/VideoUrlValidator.swift

```diff
@@ -0,0 +1,99 @@
+import Crashlytics
+import AVKit
+
+struct VideoUrlValidator {
```

Comment:
It's not just validating the URL, so this name is a bit misleading. `VideoValidator`? Does it even need to be wrapped in a struct?

Reply:
👍 good call on the name - do you have any other idea than using a struct for that? I wanted to be able to extract the logic for the validation to a separate place

Reply:
Nah, it's fine.